components/http_foundation.rst

```diff
@@ -420,12 +420,12 @@ conditional value specified in the client Request, use the
 :method:`Symfony\\Component\\HttpFoundation\\Response::isNotModified`
 method::
 
-    if ($response->isNotModified($request)) {
+    if (!$response->isNotModified($request)) {
```

Comment:
Actually, the code as it was before is correct. When using `isNotModified()` you will have other code outside of the condition that will actually build the full response in case you can not serve an empty response with a 304 status code.